zebra/rt_netlink.c

```diff
@@ -1784,7 +1784,24 @@ static ssize_t fill_seg6ipt_encap(char *buffer, size_t buflen,
 	memset(buffer, 0, buflen);
 
 	ipt = (struct seg6_iptunnel_encap *)buffer;
-	ipt->mode = SEG6_IPTUN_MODE_ENCAP;
+
+	switch (segs->encap_behavior) {
```

Comment:
The topotest and doc describe H.Encaps and H.Encaps.Reduced.
Would it be nice to add a comment about a level of support of other Behaviors just to avoid any confusion?